src/types/simulation.d.ts

```diff
@@ -28,6 +28,11 @@ type ResourceType = {
   schema: ValueSchema;
 };
 
+type ResourceSamples = {
```

Comment:
This type seems to not be used anywhere is there a reason it's included here?

Reply:
Good catch. @mattdailis you okay with me removing this?

Reply:
Looks like it's a duplicate of `ResourceType`. Removed this in the latest push.